core/src/main/scala/com/salesforce/op/stages/impl/selector/DefaultSelectorParams.scala

```diff
@@ -63,7 +63,14 @@ object DefaultSelectorParams {
   //    - "gamma"    : "inverse", "identity", "log"
   //    - "tweedie"  : power link function specified through "linkPower". The default link power in
   //    the tweedie family is 1 - variancePower.
-  val NumRound = Array(100) // number of rounds for xgboost (default 1)
-  val Eta = Array(0.1 , 0.3) // step size shrinkage for xgboost (default 0.3)
-  val MinChildWeight = Array(1.0, 5.0, 10.0) // minimum sum of instance weight needed in a child for xgboost (default 1)
+  val NumRound = Array(200) // number of rounds for xgboost (default 1)
+  val Eta = Array(0.02) // step size shrinkage for xgboost (default 0.3)
+  val MinChildWeight = Array(1.0, 10.0) // minimum sum of instance weight needed in a child for xgboost (default 1)
+  val MaxDepthXGBClassifier = Array(10) // max depth of each tree for xgboost (default is 6)
+  val MissingValPad = Array(0.toFloat) // treat 0 as missing value, (default is Float.NaN)
+  val BinaryClassXGBEvaluationMetric = Array("aucpr") // XGBoost evaluation metrics for validation data (default: error)
+  val BinaryClassXGBObjective = Array("binary:logistic") // XGBoost learning objective (default: reg:squarederror)
```

Comment:
can we add an enum for `BinaryClassXGBEvaluationMetric` and `BinaryClassXGBObjective` values?

Reply:
the grid will need to take in a `string` still, how does that work with an enum ?

Reply:
We have similar `Array[String]` in our production code for GLM hyper-parameter grid, so I think it's okay to leave this as a string instead of an enum. What do you think ?

Reply:
Ah, stupid grid. ok.